.codecov.yml

```diff
@@ -41,10 +41,11 @@ parsers:
       macro: no
 
 ignore:
-  - "bfintrinsics/**"
-  - "bfm/ioctl/arch/**"
-  - "bfvmm/include/test/**"
-  - "bfdriver/**"
+  - "bfintrinsics/**"           # Not useful to unit test
+  - "bfvmm/include/test/**"     # Don't calculate coverage on unit test code
+  - "bfdriver/**"               # Not possible to unit test (OS drivers)
```

Comment:
that comment is not true. once the ELF loader is fixed to remove shared library support, we will be able to test this again.

Reply:
I'll let you remove the comment and the exclusion once it becomes possible to do those tests in the future.

Reply:
sounds good